### PR TITLE
Update Batch Pools VMSS SKU

### PR DESCRIPTION
**Pull request by lamaks**
#### Description of changes

Update Batch Pools VMSS SKU

#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Added relevant unit test for your changes. (`yarn test`)
- [ ] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [ ] Ran precheckin (`yarn precheckin`)

### CODE REVIEW

File: packages/resource-deployment/templates/batch-account.template.json [hunk 1]
```diff
@@ -118,7 +118,7 @@
             "name": "[concat(parameters('batchAccount'), '/scan-request-pool')]",
             "dependsOn": ["[resourceId('Microsoft.Batch/batchAccounts', parameters('batchAccount'))]"],
             "properties": {
-                "vmSize": "STANDARD_A1",
```
Comment (flyingUnderTheRadar) [inline packages/resource-deployment/templates/batch-account.template.json]: does the casing matter?

Comment (flyingUnderTheRadar) [reply packages/resource-deployment/templates/batch-account.template.json]: Also do we know whether the cost changes substantially?

Comment (lamaks) [reply packages/resource-deployment/templates/batch-account.template.json]: no

Comment (lamaks) [reply packages/resource-deployment/templates/batch-account.template.json]: [Windows Virtual Machines Pricing](https://azure.microsoft.com/en-us/pricing/details/virtual-machines/windows/)

INSTANCE | VCPU | RAM | TEMPORARYSTORAGE | PAY AS YOU GO(LOW PRIORITY) | PAY AS YOU GO
-- | -- | -- | -- | -- | --
D11 v2 | 2 | 14 GiB | 100 GiB | $0.0972/hour | $0.243/hour
D1 v2 | 1 | 3.5 GiB | 50 GiB | $0.0468/hour | $0.117/hour
A1 v2 | 1 | 2 GiB | 10 GiB | $0.022/hour | $0.054/hour

Comment (lamaks) [reply packages/resource-deployment/templates/batch-account.template.json]: We cannot use A1 as per support. D11 has 2 cores that will be more efficient to run scan

**Pull request merged**